src/views/SystemWorkspace/index.tsx

```diff
@@ -0,0 +1,131 @@
+import React, { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useParams } from 'react-router-dom';
+import { useQuery } from '@apollo/client';
+import { Button, CardGroup, Grid } from '@trussworks/react-uswds';
+
+import BookmarkTag from 'components/BookmarkTag';
+import MainContent from 'components/MainContent';
+import PageHeading from 'components/PageHeading';
+import PageLoading from 'components/PageLoading';
+import SystemProfileModal from 'components/SystemProfileModal';
+import TLCTag from 'components/TLCTag';
+import GetCedarSystemBookmarksQuery from 'queries/GetCedarSystemBookmarksQuery';
+import GetSystemWorkspaceQuery from 'queries/GetSystemWorkspaceQuery';
+import { GetCedarSystemBookmarks } from 'queries/types/GetCedarSystemBookmarks';
+import {
+  GetSystemWorkspace,
+  GetSystemWorkspaceVariables
+} from 'queries/types/GetSystemWorkspace';
+import NotFound from 'views/NotFound';
+import Breadcrumbs from 'views/TechnicalAssistance/Breadcrumbs';
+
+import HelpLinks from './components/HelpLinks';
+import SpacesCard from './components/SpacesCard';
+
+export const SystemWorkspace = () => {
+  const { t } = useTranslation('systemWorkspace');
+
+  const [isSystemProfileOpen, toggleSystemProfile] = useState<boolean>(false);
+
+  const { systemId } = useParams<{
+    systemId: string;
+  }>();
+
+  const { loading, error, data } = useQuery<
+    GetSystemWorkspace,
+    GetSystemWorkspaceVariables
+  >(GetSystemWorkspaceQuery, {
+    variables: {
+      cedarSystemId: systemId
+    }
+  });
+
+  const cedarSystem = data?.cedarSystemDetails?.cedarSystem;
+  const ato = data?.cedarAuthorityToOperate[0];
```

Comment:
is `cedarAuthorityToOperate` guaranteed to have at least one element?

Reply:
This should be ok. Unavailable indices wont throw errors.